### ISSUE TITLE
[BUG] a11y: insufficient contrast in footer

### ISSUE THREAD

**Issue by jonas-hoebenreich** [formbricks/formbricks#3689]
### Issue Summary

The footer text has an insufficient color contrast of 1.53.

### Steps to Reproduce

1. Check contrast of footer

### Expected behavior

The text should have a color contrast of 4.5:1 (eg use #6e737a and the original background color: #ffffff to meet a contrast ratio of 4.78:1)

### Screenshots

![image](https://github.com/user-attachments/assets/9b8bde13-ad58-41d3-b8b1-72c518f0f0b7)

### Environment

- [X] Formbricks Cloud (app.formbricks.com)
- [ ] Self-hosted Formbricks

### Desktop (please complete the following information)

```markdown
- OS:
- Node:
- npm:
```

**Comment by jonas-hoebenreich** [comment]
I can fix this /assign

**Comment by yashkhokhar28** [comment]
I would like to work on it.

**Comment by jobenjada** [comment]
/award 150 for spotting the issue and feel free to open a PR to fix it :)
